source/simulator/parameters.cc

```diff
@@ -338,6 +321,26 @@ namespace aspect
 
     prm.enter_subsection ("Solver parameters");
     {
+      prm.enter_subsection ("GMRES parameters");
```

Comment:
Could I advocate for this to be renamed "Linear solver parameters"? The fact that we use GMRES is not really material in this regard -- it's just what we happen to use -- whereas the important fact is that it's about solving the *linear* problem.

Reply:
That makes sense.